tutorials-examples/major-updates-log.rst

```diff
@@ -43,14 +40,30 @@ All Portal, command line, and API tutorials contain the date last verified to ru
 All tutorials for the Portal aspect, for the command line (Notebook Aspect), and for the API aspect 
 are kept in the `dp0-2_lsst_io repository <https://github.com/lsst/dp0-2_lsst_io>`_.
 
-The full history for any given tutorial can be accessed via GitHub by going to the tutorial of interest
-(e.g., the `beginner Portal tutorial <https://github.com/lsst/dp0-2_lsst_io/blob/main/tutorials-examples/portal-beginner.rst>`_), 
-and clicking on "history" (near upper-right).
+The history of a given tutorial can be accessed in GitHub by going to 
+`the DP0.2 documentation repository <https://github.com/lsst/dp0-2_lsst_io>`), navigating to the file containing
+the tutorial and clicking on "history" (near upper-right).
 
 
 Major Updates Log
 =================
 
+Mar 6 2025
+----------
+
+All Jupyter Notebook tutorials were migrated to the new repository `lsst/tutorial-notebooks <https://github.com/lsst/tutorial-notebooks>`_.
+This repository no longer has a ``/data`` folder.
+Instead, input data for notebook tutorials is stored in the new Git LFS repository `lsst/tutorial-notebooks-data <https://github.com/lsst/tutorial-notebooks-data>`_.
```

Comment:
Should we also tell people the path to access the data/ directory from RSP?

Reply:
Added clarification that it is cloned into /scratch and users do not need to obtain a copy of the contents of tutorial-notebooks-data themselves.